packages/pwa-buildpack/lib/BuildBus/BuildBus.js

```diff
@@ -0,0 +1,123 @@
+const makeDebug = require('debug');
+const Tapable = require('tapable');
+const pertain = require('pertain');
+
+const VALID_HOOKS = new Set(
+    Object.values(Tapable).filter(v => typeof v === 'function')
+);
+
+class BuildBusStop {
```

Comment:
I'd love some inline documentation to understand the logic of BuildBus

Reply:
You'll get it! And this feature needs a lot more documentation before it's releasable. This PR is small and incremental, and I'm planning on adding better docs in a future release.